units/kmod-static-nodes.service.in

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Create list of static device nodes for the current kernel
+Description=Create List of Static Device Nodes for the Current Kernel
```

Comment:
Since this line is getting pretty long when using combined, maybe just drop "for the Current Kernel"?